db/database.js

```diff
@@ -17,6 +18,7 @@ const createEnrollmentStatusTable = db => {
         const insert = 'INSERT INTO enrollment_status (status) VALUES (?)';
         db.run(insert, ['enrolled']);
         db.run(insert, ['dismissed']);
+        db.run(insert, ['deferred']);
```

Comment:
Will need the complete list of statuses

Reply:
"candidate", "enrolled", "dismissed", "deferred", "transferred", "withdrew", "alum", "completed"